src/Instance/InstanceDetails/InstanceElecAccessView/InstanceElecAccessView.js

```diff
@@ -0,0 +1,107 @@
+import React, {
+  useMemo,
+} from 'react';
+import PropTypes from 'prop-types';
+import { useIntl } from 'react-intl';
+
+import {
+  Accordion,
+  MultiColumnList,
+} from '@folio/stripes/components';
+
+import {
+  checkIfArrayIsEmpty,
+} from '../../../utils';
+import {
+  noValue,
+  wrappingCell,
+} from '../../../constants';
+
+const visibleColumns = ['URL relationship', 'URI', 'Link text', 'Materials specified', 'URL public note'];
+const columnWidths = {
+  'URL relationship': '25%',
+  'URI': '25%',
+  'Link text': '25%',
+  'Materials specified': '25%',
+  'URL public note': '25%',
+};
+const getColumnMapping = intl => ({
+  'URL relationship': intl.formatMessage({ id: 'ui-inventory.URLrelationship' }),
+  'URI': intl.formatMessage({ id: 'ui-inventory.uri' }),
+  'Link text': intl.formatMessage({ id: 'ui-inventory.linkText' }),
+  'Materials specified': intl.formatMessage({ id: 'ui-inventory.materialsSpecification' }),
+  'URL public note': intl.formatMessage({ id: 'ui-inventory.urlPublicNote' }),
+});
+const getFormatter = (elAccessRelationshipsMap) => ({
+  'URL relationship': item => elAccessRelationshipsMap[item?.relationshipId] || noValue,
```

Comment:
Can NoValue component from stripes-components be used here?

Reply:
probably, I don't know what it's implemented there, just reused current implementation

Reply:
yep, I think it can be, thanks